stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -567,7 +566,7 @@ function gemm_wrapper(tA::AbstractChar, tB::AbstractChar,
     end
 end
 
-Base.@constprop :aggressive function gemm_wrapper!(C::StridedVecOrMat{T}, tA::AbstractChar, tB::AbstractChar,
+function gemm_wrapper!(C::StridedVecOrMat{T}, tA::AbstractChar, tB::AbstractChar,
```

Comment:
The other function calls are behind dynamic branches, so it doesn't seem like constants are propagated through to `matmul2x2!`, `matmul3x3!` and `_generic_matmatmul!`, unless I'm mistaken. At least, it seems that calling `gemm_wrapper!` with 2x2 arguments, the whole `matmul2x2!` gets compiled, not only one branch.